packages/block-library/src/image/edit.native.js

```diff
@@ -9,67 +9,109 @@ import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
  */
 import { MediaPlaceholder, RichText, BlockControls } from '@wordpress/editor';
 import { Toolbar, ToolbarButton } from '@wordpress/components';
+import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import ImageSize from './image-size';
 
-export default function ImageEdit( props ) {
-	const { attributes, isSelected, setAttributes } = props;
-	const { url, caption } = attributes;
+class ImageEdit extends Component {
 
-	const onUploadPress = () => {
+	constructor() {
+		super( ...arguments );
+		this.onMediaLibraryPress = this.onMediaLibraryPress.bind( this );
+		this.onUploadPress = this.onUploadPress.bind( this );
+		this.toolbarEditButton = this.toolbarEditButton.bind( this );
```

Comment:
You don't need to bind methods that will be call on this context (using `this.method()` in this module)